src/agentlab/agents/tapeagent/utils.py

```diff
@@ -0,0 +1,169 @@
+from browsergym.utils.obs import IGNORED_AXTREE_PROPERTIES, _process_bid
+
+NODES_WITH_BID = [
+    "button",
+    "link",
+    "combobox",
+    "checkbox",
+    "textbox",
+    "input",
+    "radio",
+    "menuitem",
+    "menuitemcheckbox",
+    "menuitemradio",
+    "LabelText",
+    "tab",
+]
+
+
+def flatten_axtree(
```

Comment:
Do we also have this code in TapeAgents? Or is this code somewhere in AgentLab or BrowserGym? I'm curious why we have it here and can't reuse it from some other place.

Reply:
Yes, but our current copy in the tapeagents does not work with the updated browsergym. I'm going to move that updated version into the tapeagents repo soon.

Reply:
fixed